package/rebuild-todo

```diff
@@ -219,6 +224,22 @@ read <&1
 
 pkgctl repo clone "${packages[@]}"
 
+if ((IMPORT_KEYS)); then
+	echo -e "Importing PGP keys...\n"
```

Comment:
Why are we using `echo -e "...\n"` here? 🤔 AFAIU this could just be `echo "..."` or am I missing something?

Reply:
Just style, it's basically to add a line break between "Importing PGP keys..." and the `gpg --import output`. But I have no strong feeling about this, can be removed if needed :)

Reply:
Removed